src/views/PrivacyView.vue

```diff
@@ -30,31 +30,24 @@ const navigateToHome = () => {
           </p>
           <p>E-Mail: info@remsfal.de</p>
```

Comment:
It’s not necessary to translate the text here, but it would be better to manage and maintain all text consistently in the JSON file for easier updates and uniformity.